src/app/components/client/ExposureCard.tsx

```diff
@@ -132,48 +134,74 @@ export const ExposureCard = (props: Props) => {
             detailsOpen ? styles.isOpen : ""
           }`}
         >
-          <p>
-            This site is selling and publishing{" "}
-            <a href={exposureDetailsLink}>
-              details about you.{" "}
-              <span>
-                <OpenInNew alt="" width="15" height="14" />
-              </span>
-            </a>{" "}
-            Remove this profile to protect your privacy.
-          </p>
+          {props.exposureType === "infoForSale" ? 
+          // Data broker content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-info-for-sale-part-one", 
+              {
+                elems: {"data_broker_link": <a href={props.exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt="" width="13" height="13" />
+                </span>
+              </a>
+              {". "}{l10n.getString("exposure-card-description-info-for-sale-part-two")}
+            </p>
+          </div> : 
+          // Data breach content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-data-breach-part-one", 
+              {
+                vars: {
+                  data_breach_company: props.exposureName,
+                  data_breach_date: formattedDate
+                },
+                elems: {"data_breach_link": <a href={props.exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt="" width="13" height="13" />
+                </span>
+              </a>
+              {". "}{l10n.getString("exposure-card-description-data-breach-part-two")}
+            </p>
+          </div>
+          }
           <div className={styles.exposureListOfExposureTypes}>
-            <ul>
-              <li>Your exposed info:</li>
-              <li>
+            <dl>
```

Comment:
I don't think `<dl>` is the right tag here since this isn't really a list (and we don't have nested child `<dt>` and `<dd>` elements).